antlr-kotlin-runtime/src/commonMain/kotlin/org/antlr/v4/kotlinruntime/misc/IntervalSet.kt

```diff
@@ -194,22 +194,18 @@ public class IntervalSet : IntSet {
       field = value
     }
 
+  // This is the most used constructor.
+  // Before, the vararg one was getting used, allocating an IntArray for no reason
+  public constructor()
```

Comment:
Should be the initialization move here?
If not, the other constructor does not first initialize _intervals and then override it with the received parameter?

Reply:
Thanks for spotting it! Solved, see the force-pushed change.